slave/download.py

```diff
@@ -43,11 +43,11 @@ def download_from_url(url):
 
     raise Exception("Unknown retriever")
 
-def download_for_repo(config, repo, cset="latest"):
+def download_for_repo(config, repo, cset="latest", buildtype='opt'):
```

Comment:
On IRC, we spoke that buildtype is not applicable to non-Firefox browsers and that having it as an optional parameter can make future testing of the method easier.